bigquery/google/cloud/bigquery/_helpers.py

```diff
@@ -210,11 +210,9 @@ def _row_tuple_from_json(row, schema):
 
     Note:  ``row['f']`` and ``schema`` are presumed to be of the same length.
 
-    :type row: dict
-    :param row: A JSON response row to be converted.
+        row (dict): A JSON response row to be converted.
 
-    :type schema: tuple
-    :param schema: A tuple of
+        schema (tuple): A tuple of
```

Comment:
If a docstring is multi-line, it should start on the line after the types.

```suggestion
        schema (Tuple[google.cloud.bigquery.schema.SchemaField, ...]):
            A tuple of
```

Also, we should show the type of the values contained in the tuple. See: https://docs.python.org/3/library/typing.html#typing.Tuple